pcdsdevices/pseudopos.py

```diff
@@ -39,6 +39,64 @@ def __init__(self, prefix='', parent=None, **kwargs):
             prefix = f'{parent.prefix}:{attr_name}'
 
         super().__init__(prefix=prefix, parent=parent, **kwargs)
+        self._verbose_name = verbose_name
+
+    @property
+    def calculated_dial_pos(self):
+        """
+        Calculate the dial position of the real motor dial position.
+        """
+        dial_pos = []
+        # assume the name will be something like: parent_name_self_name
+        name = self.name.split('_')[-1]
```

Comment:
Is it safe here to assume that I will always have something like this:
```
 xcs_lxt_fast.delay.name
Out[1]: 'xcs_lxt_fast_delay'
```
where the name of the pseudo single will be at the end after a `_` (in this case `delay`)

Reply:
I think `attr_name` is the correct one to use. The `_` delimiter in the name will be correct 99% of the time, but there may be situations where the underscore is ambiguous or where someone has edited the `name` attribute for some unknown reason.